modules/EnsEMBL/Web/Component/Gene/GeneSummary.pm

```diff
@@ -261,9 +261,17 @@ sub content {
       if ($attr->code eq 'artef_dupl') {
         my $label = 'Artefactural duplication';
         my $prepended_str_in_value = $label . '. ';
+
         my $text = $attr->value;
         $text =~ s/$prepended_str_in_value//g;
 
+        my $link_text = $text;
+        my $prepended_str_in_link_text = 'Real copy of this gene is ';
+        $link_text =~ s/$prepended_str_in_link_text//g;
+
+        my $full_link = sprintf('<a href="/Homo_sapiens/Gene/Summary?db=%s;g=%s;">%s</a>', $hub->param('db'), $link_text, $link_text);
```

Comment:
There should be a way to do this that doesn't need Human hardcoding

Reply:
Agreed - sorry, I missed that bit. Why is it hardcoded to human? Are these artefacts found in other species but need linking to human?

Reply:
Thanks for spotting this. No it should not be hardcoded to human. Will make the change to dynamically get the species for the link.

Reply:
Looks good!